src/timely-util/src/builder_async.rs

```diff
@@ -54,10 +54,10 @@ struct TimelyWaker {
 impl ArcWake for TimelyWaker {
     fn wake_by_ref(arc_self: &Arc<Self>) {
         // Only activate the timely operator if it's not already active to avoid an infinite loop
+        arc_self.task_ready.store(true, Ordering::SeqCst);
```

Comment:
the comment should remain above the if section. now it seems it's about setting the task_ready flag